app/grandchallenge/cases/forms.py

```diff
@@ -8,20 +8,26 @@
 from grandchallenge.uploads.models import UserUpload
 from grandchallenge.uploads.widgets import UserUploadMultipleWidget
 
-IMAGE_UPLOAD_HELP_TEXT = (
-    "The total size of all files uploaded in a single session "
-    "cannot exceed 10 GB. "
-    "A maximum of 100 files can be uploaded per session. "
-    "Please only upload one volume per session. "
-    "If your volume consists of more than this then please compress the "
-    "directory and upload a single .zip file.<br>"
-    "The following file formats are supported and will be converted to MHA: "
-    ".mha, .mhd, .raw, .zraw, .dcm, .nii, .nii.gz, .nrrd, "
-    ".fda, .fds, .tiff, .png, .jpeg and .jpg.<br>"
-    "The following file formats can be uploaded and will be converted to "
-    "TIFF: Aperio (.svs), Hamamatsu (.vms, .vmu, .ndpi), Leica (.scn), "
-    "MIRAX (.mrxs), Ventana (.bif) and DICOM-WSI (.dcm)."
-)
+IMAGE_UPLOAD_HELP_TEXT = """
+<p>
```

Comment:
I do not like all of this extra whitespace. Now you have a bunch of floating statements in the form. Before, it was clear all statements belong to the widget, now, do some belong to the widget and some to the button below? Unclear. I would prefer just to keep to line breaks if necessary. I don't think that adding empty lines will mean that users will suddenly start reading the docs.

Reply:
Check! Reverted to the line breaks, screenshots updated.